packages/cli/src/webpack/config.ts

```diff
@@ -164,7 +167,7 @@ export default async (
             plugins: [lessPluginGlob],
             paths: [
```

Comment:
Maybe it would be better to add an option to override all paths, instead of just one. It would bring more flexibility.